CHANGELOG.md

```diff
@@ -2,12 +2,17 @@
 
 ## Unreleased
 
-* Add overloaded functions with callbacks for `start` and `vault` methods in
-  `PayPalWebCheckoutClient`
-* Modify `PayPalWebCheckoutRequest` to include `appSwitchWhenEligible` property to control app
+* Adds new property `appSwitchWhenEligible`in `PayPalWebCheckoutRequest` to include to control app
   switch behavior
-* Breaking Changes
-  * Make start and vault functions in `PayPalWebCheckoutClient` suspend functions
+* Adds new property `appSwitchWhenEligible`in `PayPalWebVaultRequest` to include to control app
```

Comment:
Can we remove these line breaks? The Markdown renderer on GitHub.com will word wrap for us.